solutions/standard/variables.tf

```diff
@@ -0,0 +1,218 @@
+  }
+}
+
+variable "image_name" {
+   description = "Enter a valid image name for Wazi VSI"
+   type        = string
+   default     = "ibm-zos-3-1-s390x-dev-test-wazi-1"
+}
+
+variable "override" {
+description = "Override default values with custom JSON template. This uses the file `override.json` to allow users to create a fully customized environment."
+type        = bool
+default     = true
+}
+
+variable "sm_service_plan" {
+  type        = string
+  description = "Type of service plan to use to provision Secrets Manager if not using an existing one."
+  default     = "standard"
+}
+
+variable "root_ca_name" {
+  type        = string
+  description = "Name of the Root CA to create for a private_cert secret engine. Only used when var.existing_sm_instance_guid is false"
+  default     = "root-ca"
+}
+
+variable "intermediate_ca_name" {
+  type        = string
+  description = "Name of the Intermediate CA to create for a private_cert secret engine. Only used when var.existing_sm_instance_guid is false"
+  default     = "intermediate-ca"
+}
+
+variable "certificate_template_name" {
+  type        = string
+  description = "Name of the Certificate Template to create for a private_cert secret engine. When var.existing_sm_instance_guid is true, then it has to be the existing template name that exists in the private cert engine."
+  default     = "my-template"
+}
+
+variable "create_policy" {
+  description = "Set to true to create a new access group (using the value of var.access_group_name) with a VPN Client role"
+  type        = bool
+  default     = true
+}
+
+variable "vpn_client_access_group_users" {
+  description = "List of users in the Client to Site VPN Access Group"
+  type        = list(string)
+  default     = []
+}
+
+variable "access_group_name" {
+  type        = string
+  description = "Name of the IAM Access Group to create if var.create_policy is true"
+  default     = "client-to-site-vpn-access-group"
+}
+
+variable "vpn_server_routes" {
+  type = map(object({
+    destination = string
+    action      = string
+  }))
+  description = "Map of server routes to be added to created VPN server."
+  default = {
+    "vpn-ibm-network" = {
+      destination = "10.0.0.0/8"
+      action      = "translate"
+    }
+    "route-vpn-2-services" = {
+      destination = "166.9.0.0/16"
+      action      = "translate"
+    }
+    "route-vpn-2-dns" = {
+      destination = "161.26.0.0/24"
+      action      = "translate"
+    }
+  }
+}
+
+variable "root_ca_max_ttl" {
+  type        = string
+  description = "Maximum TTL value for the root CA"
+  default     = "8760h"
+}
+
+variable "root_ca_common_name" {
+  type        = string
+  description = "Fully qualified domain name or host domain name for the certificate to be created"
+  default     = "cloud.ibm.com"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "Name of the resource group to use for this example. If not set, a resource group is created."
```

Comment:
The wording is wrong here, this is a solution, not an example. Either way, Landing zone will create resource groups by default. Are you not going to use one of those?